main.go

```diff
@@ -90,6 +92,17 @@ func main() {
 	}
 }
 
+func readPassword(prompt string) (string, error) {
+	state, err := terminal.MakeRaw(int(os.Stdin.Fd()))
+	if err != nil {
+		return "", err
+	}
+	defer terminal.Restore(int(os.Stdin.Fd()), state)
```

Comment:
There's some weird cases /dev/tty doesn't work I've run into:  Chroots without a /dev, etc, so if we do want to go that route we should consider having a fallback.